ui/src/app/components/side-nav.tsx

```diff
@@ -285,22 +286,22 @@ export const SideNav = (props: SideNavProps) => {
         onToggleSideNav={() => onToggleSideNav()}
         href={'/workspaces'}
         active={workspacesActive()}
-        disabled={!hasRegisteredAccess(profile.accessTierShortNames)}
```

Comment:
changed the interface to Profile so the caller doesn't need to know what *part* of the Profile provides this information.